src/frontend/apps/desk/src/features/teams/components/PanelTeams.tsx

```diff
@@ -0,0 +1,129 @@
+import { Loader } from '@openfun/cunningham-react';
+import React, { Fragment, useEffect, useMemo, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Box, Text } from '@/components';
+import { InfiniteScroll } from '@/components/InfiniteScroll';
+
+import { TeamResponse, useTeams } from '../api/useTeams';
+import { useTeamStore } from '../store/useTeamsStore';
+
+import { PanelTeam } from './PanelTeam';
+
+type DictTeams = {
+  [page: string]: TeamResponse[];
+};
+
+export const PanelTeams = () => {
+  const { t } = useTranslation();
+
+  const [dictTeams, setDictTeams] = useState<DictTeams>({});
+  const [page, setPage] = useState(1);
+  const ordering = useTeamStore((state) => state.ordering);
+  const { data, isError, isLoading } = useTeams({
+    page,
+    ordering,
+  });
+  const containerRef = useRef<HTMLDivElement>(null);
+  const scrollLoading = useRef(false);
+  const teams = useMemo(() => {
+    let teams: TeamResponse[] = [];
+    Object.keys(dictTeams)
+      .sort()
+      .forEach((key) => (teams = teams.concat(dictTeams[key])));
```

Comment:
I think this logic would be refactored with the infinite query hook, but let's discuss it orally

Reply:
https://github.com/numerique-gouv/people/pull/31/commits/ccef7d82b26ff1a60c90314be58c89d48aa38580